### ISSUE TITLE
Country not known

### ISSUE THREAD

**Issue by YvonneBlaine** [teuteuf/worldle#108]
A couple of days ago I was looking for a location south of Ireland and I tried using Wales to get a little closer and check direction.  I got a notice that it was UNKNOWN.  I was surprised, of course, and checked Google to see if it has more to its name, like "Kingdom of" but it appears to be just Wales.

**Comment by teuteuf** [comment]
Same than for Scotland, you can check the FAQ for details about it. :)

**Comment by YvonneBlaine** [comment]
Thank you for the answer.  When I first tried checking The FAQ they
wouldn't open up to give the answer.  It's possible I was clicking in the
wrong spot but I thought it may be because I hadn't actually set up an
account, yet?

I have now opened an account and I was able to see the answer.  Or it's
just that I hit the right spot when looking.

Also, recently I've become aware that President James Madison's home is
called Montpelier. Since I know the capital of Vermont is also Montpelier,
it occurred to me that there must be a Montpelier somewhere in Europe.  I
googled to see where the name came from and it said a medieval city in
France, Montpellier. 😃

It looks like a fabulous city.

Once again, thanks for the answer.

Sue
